aws_jobs/cyhy-data-extract.py

```diff
@@ -138,12 +138,13 @@ def cleanup_bucket_files(object_retention_days):
                 ]
             },
         )
-        for err in del_resp["Errors"]:
-            sys.stderr.write(
-                "Error: {} when deleting {} - {}\n".format(
-                    err["Message"], err["Key"], err["Code"]
+        if "Errors" in del_resp:
+            for err in del_resp["Errors"]:
```

Comment:
An alternative way to do this would be:
`for err in del_resp.get("Errors", []):`
which eliminates the need for the `if` statement